alert-database/src/test/java/com/synopsys/integration/alert/database/api/mock/MockSettingsKeyRepository.java

```diff
@@ -0,0 +1,152 @@
+package com.synopsys.integration.alert.database.api.mock;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import com.synopsys.integration.alert.database.settings.SettingsKeyEntity;
+import com.synopsys.integration.alert.database.settings.SettingsKeyRepository;
+
+public class MockSettingsKeyRepository implements SettingsKeyRepository {
+
+    Map<String, SettingsKeyEntity> settingsKeyEntities = new HashMap<>();
```

Comment:
I understand this is a Mock object for tests but I think it would be best to follow good conventions and make these fields private and create setters or some methods to add to these Map's.